docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,281 @@
+```yaml
+- &Org1
+
+  ...
+
+  Policies:
+      Readers:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin', 'Org1MSP.peer', 'Org1MSP.client')"
+      Writers:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin', 'Org1MSP.client')"
+      Admins:
+          Type: Signature
+          Rule: "OR('Org1MSP.admin')"
+      Endorsement:
+          Type: Signature
+          Rule: "OR('Org1MSP.peer')"
+```
+
+Org1 is associated with four sets of signature policies. The policies can only be satisfied by signatures from Org1. However, each policy lists a different set of roles from within the organization that satisfy the policy. The `Admins` policy can only be satisfied by the transactions submitted by an admin identity, while only peers can satisfy the `Endorsement` policy. Each transaction can satisfy multiple policies. For example, if a transaction was endorsed by a peer from Org1 and Org2, the transaction would satisfy the `Endorsement:` signature policy associated with both organizations.
+
+The signature polices for each organization are read by ImplicitMeta policies at a higher levels of the channel configuration. Instead of directly reading the signatures that are submitted to the channel, ImplicitMeta policies reads other policies in the channel configuration. An ImplicitMeta policies is satisfied if a transaction satisfies the referenced signature policies.
+
+You can find an example of ImplicitMeta policies from the **Application** section of `configtx.yaml` file:
+```yaml
+Policies:
+    Readers:
+        Type: ImplicitMeta
+        Rule: "ANY Readers"
+    Writers:
+        Type: ImplicitMeta
+        Rule: "ANY Writers"
+    Admins:
+        Type: ImplicitMeta
+        Rule: "MAJORITY Admins"
+    LifecycleEndorsement:
+        Type: ImplicitMeta
+        Rule: "MAJORITY Endorsement"
+    Endorsement:
+        Type: ImplicitMeta
+        Rule: "MAJORITY Endorsement"
+```
+
+The ImplicitMeta policies in the **Application** section govern how peer organizations interact with the channel. Each policy references the signature policies associated with each channel member. You see the relationship between the policies in the **Application** section and the policies in the **Organization** section below:
+
+  ![Application policies](application-policies.png)  
+
+*Figure 1: The Admins ImplicitMeta policy is satisfied the Admins signature policy associated by each organization.*
+
+Each policy is referred to its path in the channel configuration. Because the policies in the **Application** section are located in the application group, which is located inside the channel group, they are referred to as `Channel/Application` policies. Since the most places in the Fabric documentation refer to policies by their path, we will refer to policies by their path from now on,
+
```

Comment:
from now on.

Reply:
Since most places in the Fabric documentation refer to policies by their path, from now on, we will refer to policies by their path.